releases/v20.2.11.md

```diff
@@ -0,0 +1,128 @@
+- Fixed a bug where empty [zone configurations](../v20.2/configure-zone.html) get created for certain indexes during [`ALTER PRIMARY KEY`](../v20.2/alter-primary-key.html). [#65175][#65175]
+- Fixed a bug causing [`SHOW CREATE TABLE`](../v20.2/show-create.html) output to not display the [zone configurations](../v20.2/configure-zone.html) of a table or index if there were no partitions, even if there were zone configurations on the index or table. [#65175][#65175]
+- Fixed a bug where [`ALTER DATABASE ... SET OWNER ...`](../v20.2/alter-database.html) did not work if the database name was a keyword. [#65367][#65367]
+- Fixed a bug where [`SHOW CREATE TABLE`](../v20.2/show-create.html) would show the [zone configurations](../v20.2/configure-zone.html) of a table with the same name from a different schema. [#65369][#65369]
+- Previously, CockroachDB would crash when attempting to create a table using [`CREATE TABLE ... AS`](../v20.2/create-table-as.html) syntax where `AS` selects from `crdb_internal.node_statement_statistics`, `crdb_internal.node_transaction_statistics`, or `crdb_internal.node_txn_stats` virtual tables. [#65544][#65544]
+- Fixed a bug where binary [`TIMETZ`](../v20.2/time.html) values were not being decoded correctly when sent as a parameter in the wire protocol. [#65579][#65579]
+- A certain percentage of cases in which a node could have served a follower read were not handled correctly, resulting in the node routing the request to another nearby node for no reason. This has been fixed. [#65470][#65470]
+- Fixed a race condition during transaction cleanup that could leave old transaction records behind until MVCC garbage collection. [#65384][#65384]
+- Improved transaction cleanup for disconnected clients to reduce intent buildup. [#65384][#65384]
+- [Scheduled backups](../v20.2/manage-a-backup-schedule.html) with [interleaved tables](../v20.2/interleave-in-parent.html) can now be created with the `include_deprecated_interleaves` option. [#65730][#65730]
+- Calling [`get_bit` or `set_bit`](../v20.2/functions-and-operators.html#string-and-byte-functions) on a byte array argument now goes to the correct index of the underlying bitstring, which matches the Postgres behavior. [#65787][#65787]
+- Previously, [`ALTER DATABASE ... CONVERT TO SCHEMA`](../v20.2/alter-database.html) could potentially leave the schema with invalid privileges thus causing the privilege descriptor to be invalid. [#65813][#65813]
+- Fixed a scenario in which a rapid sequence of splits could trigger a storm of Raft snapshots. This would be accompanied by log messages of the form "would have dropped incoming MsgApp, but allowing due to ...", which tended to occur as part of [`RESTORE`](../v20.2/restore.html)/[`IMPORT`](../v20.2/import.html) operations. [#65500][#65500]
```

Comment:
```suggestion
- Fixed a scenario in which a rapid sequence of splits could trigger a storm of Raft snapshots. This would be accompanied by log messages of the form `would have dropped incoming MsgApp, but allowing due to ...`, which tended to occur as part of [`RESTORE`](../v20.2/restore.html)/[`IMPORT`](../v20.2/import.html) operations. [#65500][#65500]
```